### ISSUE TITLE
NMRangeSlider with Swift 

### ISSUE THREAD

**Issue by rawanOmari** [muZZkat/NMRangeSlider#58]
hi :) 
i'm trying to use NMRangeSlider with my projject written in Swift. 
i faced a problem:  the slider doesn't appear at all! 
when i entered the Debug View Hierarchy and see if it's on canvas, the slider was there !!!
![screen shot 2015-07-02 at 12 48 32 pm](https://cloud.githubusercontent.com/assets/12219139/8475614/1faadb7c-20c2-11e5-8217-1671ff5416cc.png)
but all its component are white, so it seemed that the slider doesn't take the default values. 
So my question is : 
How to deal with NMRangeSlider with swift ? 
why this problem happened?

**Comment by bre7** [comment]
Set this commit in your Podfile:

``` ruby
pod 'NMRangeSlider',                :git => 'https://github.com/chanil1218/NMRangeSlider.git', :commit => 'c89890c'
```

**Comment by dnsmob** [comment]
humm.. tried that, doesnt seem to work. more ideas?

**Comment by bre7** [comment]
The slider or the knobs aren't visible ?

**Comment by dnsmob** [comment]
exactly. debugging the view i can see the object is there, but seems to be transparent or something.. i added images into the xcassets, project.. but nothing seems to make a difference. i added logs all over the place and all methods seem to be called all the way to the addSubview. so no ideas what's going on -- it works fine on a pure objc proj though.

**Comment by bre7** [comment]
I have it working on a Swift project using that commit that fixes the bundle bug.

**Comment by dnsmob** [comment]
maybe i'm doing something wrong then.. would you share a working empty project please?

**Comment by bre7** [comment]
Sure, np: https://github.com/bre7/NMRangeSliderTest

**Comment by hrodriguezfigueroa** [comment]
@bre7 That works :) thanks

**Comment by xsahoo** [comment]
@bre7 could you repost your working project? I'm having the same issue

**Comment by bre7** [comment]
You just have to use a specific commit in your Podfile (until #54 is merged):

``` ruby
pod 'NMRangeSlider',                :git => 'https://github.com/chanil1218/NMRangeSlider.git', :commit => 'c89890c'
```

**Comment by xsahoo** [comment]
@bre7 That did the trick, thanks!
